core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -993,6 +993,12 @@ class DAGScheduler(
           JavaUtils.bufferToArray(closureSerializer.serialize((stage.rdd, stage.func): AnyRef))
       }
 
+      if (taskBinaryBytes.length > TaskSetManager.TASK_SIZE_TO_WARN_KB * 1024) {
```

Comment:
Would you mind putting this in a different PR? This change -- adding an extra log message -- is potentially helpful but unrelated to this pull request